src/playground/dist/app.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
This looks like compiled version of [src/playground/src/app.ts ](https://github.com/xFAANG/askql/pull/196/files#diff-231aef61cb7355918bb7bcaee3989cff)

Reply:
You're right, I didn't notice including `dist/` folder. Removing it now.